src/index.js

```diff
@@ -51,15 +53,22 @@ function generateCurrentDate() {
   let day = rawDate.getDate();
   if (day < 10) {
     day = `0${day.toString()}`
-  };
+  }
   let month = rawDate.getMonth() + 1;
   if (month < 10) {
     month = `0${month.toString()}`
-  };
+  }
   const year = rawDate.getFullYear();
   return `${year}/${month}/${day}`
 }
 
+function analyzeManagerClick(event) {
+  event.preventDefault(); 
+  if (event.target.classList.contains('search-submit')) {
```

Comment:
When I click search with a valid user name in there (copied right from the users data, case sensitive) it takes me back to the login screen

Reply:
@ErinUntermeyer Hmm, I've tested this a bunch & just tested again, and am not getting this error. Can you give me any more details?

Reply:
I just entered in two different customer names (Leatha Ullrich and Rocio Schuster) and when I clicked 'Search' it took me to the main login screen again.

<img src="http://g.recordit.co/8bN5y6d1kB.gif" alt="Refactor Tractor: Fit-Lit main page" height=auto width=75%/>

Reply:
@ErinUntermeyer I see when you log in, the "Tom Smith" info is there - that was the placeholder info I had in there before I made the changes in this PR, so I'm guessing you somehow pulled down an outdated branch!